src/CommunicationLayer/CommDeviceControl/CommDeviceManager.cpp

```diff
@@ -43,3 +59,11 @@ void CommDeviceManager::handleUdpDataIncoming()
         }
     }
 }
+
+void CommDeviceManager::handleJsonDataIncoming(QByteArray data)
+{
+    if (!data.isEmpty())
```

Comment:
we should probably keep a warning message if it is empty, instead of silently doing nothing